Gruntfile.js

```diff
@@ -204,6 +204,7 @@ module.exports = function(grunt) {
     'i18nextract',
     'copy:addBackLanguageKey',
     'json_remove_fields',
+    'json_remove_fields',
```

Comment:
OK this is terrible but it worked?????? will you allow it @james-wills-sf 😬

Reply:
Omg. Is it because it's a nested json object?

It's fine with me, can you add a comment above that line explaining that it's necessary?

Reply:
no idea tbh. just copied this technique from our weird hacked fork of that package: https://github.com/SFDigitalServices/grunt-json-remove-fields/commit/b97bb836aaf05556336ece133193bda211f1b903#diff-3dbc63a47e43d636d8df67c66371b031R57. I'll add a comment.

Reply:
I think it is because it's nested. and I have 3 levels of missing keys? and they only run it twice from that comment? Seems pretty janky all around.

Reply:
comment added. going to go ahead and merge now 🤷